src/transport/CASESession.h

```diff
@@ -37,7 +39,29 @@
 
 namespace chip {
 
+constexpr uint16_t kAEADKeySize = 16;
+
+constexpr uint16_t kSigmaParamRandomNumberSize = 32;
+constexpr uint16_t kTrustedRootIdSize          = 20;
+constexpr uint16_t kMaxTrustedRootIds          = 5;
+
+constexpr uint16_t kIPKSize = 0;
+
 using namespace Crypto;
+using namespace Credentials;
+
+struct CASESessionSerialized;
+
+struct CASESessionSerializable
+{
+    uint16_t mI2RR2IKeyLen;
+    uint8_t mI2RR2IKey[kAEADKeySize * 2];
+    uint8_t mPairingComplete;
```

Comment:
this is named like a bool, but stored as uint8_t.can it be a bool?

Can the items below be chip::NodeId (and some keyid for key indices) for clarity?

Reply:
I guess the idea here is to fix all the class' data types to a certain fixed length value in bytes. This is just mirroring what has been done in PASESession (with bool types).
Yes, I agree replacing uint64_t with NodeID for clarity